### PR TITLE
Remove non-root suffix to tags

### PR DESCRIPTION
**Pull request by HenryNguyen5**
Since we default to non-root images now, the suffix has been removed for
them. For root images, we publish a latest tag for root images with a
-root suffix now.

### CODE REVIEW

File: .github/actions/build-sign-publish-chainlink/action.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ runs:
     uses: docker/metadata-action@e5622373a38e60fb6d795a4421e56882f2d7a681 # v3.6.2
     with:
       flavor: |
-        suffix=-root
```
Comment (chainchad) [inline .github/actions/build-sign-publish-chainlink/action.yml:116]: Do we want this here?

```
latest=false
```

File: .github/actions/build-sign-publish-chainlink/action.yml [hunk 2]
```diff
@@ -145,7 +145,7 @@ runs:
       flavor: |
         latest=auto
```
Comment (chainchad) [inline .github/actions/build-sign-publish-chainlink/action.yml:146]: Should this also be set to the following?

```
latest=false
```

Comment (HenryNguyen5) [reply .github/actions/build-sign-publish-chainlink/action.yml:146]: Nah, so we'll keep publishing a latest tag in ECR, could still have some use anyway. Unless you feel strongly about removing latest altogether

File: .github/actions/build-sign-publish-chainlink/action.yml (hunk 1 above)
Comment (HenryNguyen5) [reply .github/actions/build-sign-publish-chainlink/action.yml:116]: This will publish `latest-root` instead, dont think we should remove it, should we?

Comment (chainchad) [reply .github/actions/build-sign-publish-chainlink/action.yml:116]: Nah, let's keep it then 👍

**Review by chainchad — APPROVED** [review]

**Pull request merged**